mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+  /**
+   * Checkbox status
+   */
+  checked: boolean;
```

Comment:
lets start all boolean with prefix `is`

Reply:
I propose to encapsulate the checkbox state inside (as I seen you propose to handle creating additional state outside each time i want to use the checkbox. looks like wrong approach)

Reply:
rewrote component with encapsulate  state